lib/naver/searchad/api/adgroup/service.rb

```diff
@@ -5,35 +5,55 @@ module Adgroup
         class Service < Naver::Searchad::Api::Core::BaseService
 
           def initialize
-            super('https://api.naver.com/', 'ncc/adgroups/')
+            super('https://api.naver.com/', 'ncc/')
```

Comment:
Oh yeah I mean, it's because it's hardcoded that we should at least abstract it (like for your specs :)